services/blockstorage/test/transaction_receipt.go

```diff
@@ -0,0 +1,82 @@
+package test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/orbs-network/orbs-network-go/crypto/digest"
+	"github.com/orbs-network/orbs-network-go/test/builders"
+	"github.com/orbs-network/orbs-spec/types/go/primitives"
+	"github.com/orbs-network/orbs-spec/types/go/services"
+	"time"
+)
+
+var _ = Describe("Block Storage", func() {
+	When("fetching transaction receipts", func() {
```

Comment:
please migrate to testify

Reply:
after this is approved **not** as part of this PR
see https://tree.taiga.io/project/orbs-network/us/108 ETA is this week